core/src/main/java/io/questdb/std/Files.java

```diff
@@ -42,7 +42,11 @@ public final class Files {
     public static final int MAP_RO = 1;
     public static final int MAP_RW = 2;
     public static final char SEPARATOR;
-
+    public static final int POSIX_FADV_SEQUENTIAL;
+    public static final int POSIX_FADV_RANDOM;
+    public static final int FILES_RENAME_ERR_OK = 0;
```

Comment:
nit: `FILES_RENAME_ERR_OK` reads weird considering that it's a non-error response. I'd rename it to `FILES_RENAME_OK`.

Reply:
Addressed in 8d41fbd912061a6d3a162aa2aba381c3ceef7f6f